csm_web/scheduler/management/commands/export_section.py

```diff
@@ -11,14 +11,12 @@ def add_arguments(self, parser):
 
     def handle(self, *args, **options):
         for email in options["emails"]:
-            profs = Profile.objects.filter(user__email=email).exclude(
-                role=Profile.STUDENT
-            )
+            profs = Mentor.objects.filter(user__email=email)
             self.stdout.write("====SECTIONS FOR MENTOR {}====".format(email))
```

Comment:
```suggestion
            self.stdout.write(f"====SECTIONS FOR MENTOR {email}====")
```
Just to be consistent with the rest of the code base